code/monitor/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>monitor</name>
```

Comment:
I would choose a more representative name, as this could easily have conflicts with other packages
`monitor_backend_tutorial` seems better for me.

Reply:
`monitor_tutorial` finally chosen